### ISSUE TITLE
RGB effects can be enabled in Tachyon mode with Tab+Enter

### ISSUE THREAD

**Issue by mikey-dg** [WootingKb/wootility-issues#126]
It is possible to enable any RGB effect while Tachyon mode is enabled by tabbing through UI elements and pressing enter.  Additionally, no Tab focus indicator is shown for any buttons or controls in the application except for text boxes.

**To Reproduce**
1. Ensure Tachyon mode is enabled, and no RGB effects are on
2. Close and re-run Wootility, wait for it to finish connecting to the device
3. Press Tab 10-20 times to select an RGB effect
4. Press Enter

**Wootility Version**
v4.2.2 and v.4.3.4

**OS**
Windows 7

I have no idea if this affects the function of the keyboard in any meaningful way, but it did cause the previous version of Wootility to have a bit of a meltdown.  v4.3.4 seems to take it in stride.

While keyboard may not be the preferred method for navigating the Wootility, as long as it accepts keyboard input there should probably be some sort of focus indicator.  Currently, Tab focus is only shown on text boxes.  I found this bug while testing keys, thinking it accepted no other keyboard input due of the lack of visual feedback!

**Comment by simon-wh** [comment]
This has been fixed and will be included in the next Wootility update
